packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -153,11 +153,26 @@ function OrgNav({
 }: OrgNavProps) {
   const [accessManagerOpen, setAccessManagerOpen] = useState(false);
   const [onPremMenuOpen, setOnPremMenuOpen] = useState(false);
-  const { width: viewportWidth } = useViewportSize();
   const onElementClick = useOnElementClick();
-  const { orgNav } = urls;
-  const isTablet = viewportWidth < breakpoints.medium;
-  const isMobile = viewportWidth < breakpoints.small;
+  const viewportSize = useViewportSize();
+  const [showProductTour, setShowProductTour] = useState(false);
+
+  const isTablet = viewportSize
```

Comment:
Could we do this rather than the ternary?

```js
const isTablet = viewportSize && viewportSize.width < breakpoints.medium;
```

Reply:
sure, though I think we need `!!` in front of `viewportSize` so that `isTablet` becomes `false` instead of `null`

Reply:
Is there a case where useViewportSize returns null?

Reply:
yeah, on initial render, since the initial read was moved into an effect